tools/db_crashtest.py

```diff
@@ -92,7 +92,7 @@
         [t * 16384 if t < 3 else 1024 * 1024 * 1024 for t in range(1, 30)]),
     # Sync mode might make test runs slower so running it in a smaller chance
     "sync" : lambda : random.choice(
-        [0 if t == 0 else 1 for t in range(1, 20)]),
+        [1 if t == 0 else 0 for t in range(0, 19)]),
```

Comment:
`range(0, 20)` to include 0, 1, ..., 19?

Reply:
Oh right. I'll change it.